ComfyUI/Dockerfile.local-cpu

```
@@ -80,8 +80,24 @@ skip_migration_check = True
 model_download_by_agent = False
 EOF
 
+# create changeset saving script
+RUN mkdir changeset
+RUN <<EOF cat >> save-changeset.sh
+#!/bin/bash
+git add --all
+git commit -m 'All changes'
+git archive -o changeset/update.tar.gz HEAD
+EOF
+RUN chmod +x save-changeset.sh
```

Hadolint (2.12.0)

[error] unexpected 'R'
expecting a new line followed by the next instruction

(DL1000)